src/utilities.jl

```diff
@@ -66,11 +53,79 @@ function get_cache_entry(word, server, modules=[])
     return entry
 end
 
+function unpack_dot(id, args = Symbol[])
+    if id isa Expr && id.head == :. && id.args[2] isa QuoteNode
+        if id.args[2].value isa Symbol
+            unshift!(args, id.args[2].value)
+            unpack_dot(id.args[1], args)
+        else
+            return Symbol[]
+        end
+    elseif id isa Symbol
+        unshift!(args, id)
+    else
+        return Symbol[]
+    end
+    return args
+end
+
+
+repack_dot(args::Symbol) = args
+function repack_dot(args::Vector)
+    if length(args) == 1
+        return first(args)
+    else
+        return repack_dot([Expr(:., first(args), QuoteNode(args[2])); args[3:end]])
+    end
+end
+
+function get_cache_entry(id, server, modules = [])
+    ids = unpack_dot(id)
+    if !isempty(ids)
+        for m in vcat([:Base, :Core], modules)
+            if m in keys(server.cache)
+                if first(ids) == m
+                    shift!(ids)
+                    return get_cache_entry(ids, server.cache[m])
+                elseif first(ids) in server.cache[m][:EXPORTEDNAMES]
+                    return get_cache_entry(ids, server.cache[m])
+                end
+            end
+        end
+    end
+    return entry = EmptyCacheEntry
+end
+
+function get_cache_entry(ids::Vector{Symbol}, cache::Dict)
+    if isempty(ids)
+        return entry = EmptyCacheEntry
+    end
+    for (k, entry) in cache
+        if k == first(ids)
+            if length(ids) == 1
+                if entry isa Dict
+                    return (k, "Module", []) 
+                else
+                    return entry
+                end
+            else
+                if entry isa Dict
+                    shift!(ids)
+                    return get_cache_entry(ids, entry)
+                else
+                    return entry = EmptyCacheEntry
+                end
+            end
+        end
+    end
+    return entry = EmptyCacheEntry
+end
+
 function uri2filepath(uri::AbstractString)
     uri_path = normpath(unescape(URI(uri).path))
 
     if is_windows()
-        if uri_path[1]=='\\'
+        if uri_path[1] == '\\'
```

Comment:
Can you change this to ``if uri_path[1] == '\\' || uri_path[1] == '/'``. For some reason we now get a ``/`` at the beginning of the path on Windows, which we don't need.